src/Data/Quantity/Math.purs

```diff
@@ -178,6 +179,11 @@ mean xs = (_ ⊘ n) <$> foldM (⊕) (head xs) (tail xs)
   where
     n = scalar' (Decimal.fromInt (length xs))
 
+geomean ∷ NonEmptyList Quantity → Result
+geomean xs = (_ `pow` (Decimal.recip n)) <$> foldM (⊗) (head xs) (tail xs)
```

Comment:
It's now failing with "Unknown value Decimal.recip". Can you try
```suggestion
geomean xs = (_ `pow` (Decimal.fromInt 1 / n)) <$> foldM (⊗) (head xs) (tail xs)
```

Reply:
I fixed that a few commits back I think? The current error is:

```
  Could not match type
            
    Quantity
            
  with type
               
    t2 Quantity
               

while checking that type Quantity -> Quantity -> Quantity
  is at least as general as type t0 -> t1 -> t2 t0
while checking that expression qMultiply
  has type t0 -> t1 -> t2 t0
in value declaration geomean

where t2 is an unknown type
      t1 is an unknown type
      t0 is an unknown type

```

Reply:
(I used `one` but that's the same thing it seems)